app.js

```diff
@@ -3,16 +3,18 @@ import express from 'express'
 import nodeCleanup from 'node-cleanup'
 import routes from './routes.js'
 import { init, cleanup } from './whatsapp.js'
+import cors from 'cors'
 
 const app = express()
 const host = process.env.HOST ?? '127.0.0.1'
-const port = parseInt(process.env.PORT ?? 8000)
+const port = parseInt(process.env.PORT ?? 3333)
```

Comment:
I want you leave this just like how it previously is. If you want to use custom port, just edit from the `env` file instead.

Reply:
ok 👍 i will keep 8000 as default port